docs/channel.md

```diff
@@ -0,0 +1,114 @@
+title: Payment Channels
+description: 
+---
+# Payment Channels
+
+A library for building payment channel smart contracts.
```

Comment:
This looks too much like a readme, its style is quite different from other docs I feel